README.md

```diff
@@ -4,7 +4,7 @@
 | [Annyce Davis](https://twitter.com/brwngrldev) | Laurel, MD | [http://www.adavis.info/p/public-speaking.html](http://www.adavis.info/p/public-speaking.html) |
 | [Audrey Troutt](https://twitter.com/auditty) | Philadelphia, PA | [http://audreytroutt.com/#speaking](http://audreytroutt.com/#speaking) |
 | [Brenda Cook](https://twitter.com/kenodoggy) | Denver, CO | [http://kenodoggy.com](http://kenodoggy.com) |
-| [Caren Chang](https://www.linkedin.com/in/carenchang) | San Francisco, CA ||
+| [Caren Chang](https://www.linkedin.com/in/carenchang) | San Francisco, CA |[http://www.carenslaboratory.com/speaking.html](http://www.carenslaboratory.com/speaking.html)|
```

Comment:
Curious why you prefer to list LinkedIn instead of Twitter?

Reply:
i think i submitted my info when i wasn't really using twitter. i shall update it